Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -454,7 +454,8 @@ import BraintreeDataCollector
                         )
                         return
                     }
-                    self.launchPayPalApp(with: url, completion: completion)
+                    let merchantAccountId = json["merchantId"].asString()
```

Comment:
To match style guide:
```suggestion
                    let merchantAccountID = json["merchantId"].asString()
```